invoicer_project/static/js/add_item_request.js

```diff
@@ -17,23 +17,25 @@ import {
     validatePrice,
     validationDropdown,
     validateAmountInStock,
-    validateBarcode
+    validateBarcode, setErrorAttributeToDropdown,
 } from './validation_utils.js'
 import {actionBasedOnStatusCode, obtainUserInitials, sendAddEditRequest} from './request_utils.js'
 import {
+    countryField,
     hideUnnecessaryElementsInMenu,
 } from './utils_clients.js'
+import {removeStylesFromDropdownElements} from "./dropdown.js";
 
 
-function validateClientAdd() {
+function validateItemAdd() {
     clearErrorAttributes(returnAllFields());
     clearErrorAttributes(additionalFieldsContainer);
     clearErrorAttributes(amountAdditionalFieldsContainer);
     return [
         validateName(nameField.value),
         validatePrice(priceField.value),
-        validationDropdown("currency"),
-        validationDropdown("basic_unit"),
+        validationDropdown("currency_input_dropdown"),
```

Comment:
This is just comment. I will recommend to you look up design pattern Chain of Responsibility. It great fits for complex validation cases like this.

Reply:
I will read about it. Thanks!